packages/vapor/scss/controls/radios.scss

```
@@ -122,6 +122,50 @@
     }
 }
 
+.radio-card-container {
+    --shadow-color: rgba(229, 232, 232, 0.75);
+    flex: 0;
+
+    .radio-card {
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        padding: 20px;
+        border: $default-border;
+        border-radius: 8px;
+        box-shadow: 0px 2px 8px var(--shadow-color);
+        transition: all 0.3s ease-in-out;
```

If you add the [`card` class](https://github.com/coveo/react-vapor/blob/master/packages/vapor/scss/components/card.scss#L17-L23) to the RadioCard component, we could ditch some of the redundant style here.

```suggestion
    .radio-card {
        display: flex;
        flex-direction: column;
        height: 100%;
        padding: 20px;
        transition: all 0.3s ease-in-out;
```

---

That's what i did at first but because of how sass resolves extensions, it was hard to get the order of precedence I wanted for the hover effect to work properly.

---

I was using @extends though so maybe if i just add the class to the element it would work

---

It does!